### ISSUE TITLE
Replication is not working

### ISSUE THREAD

**Issue by the100rabh** [sapo/kyoto#8]
Create master Node for Kyoto Tycoon with file database
Start pushing data to it 
After around 10 Million keys are pushed to it, start a slave to this master node
Slave does not sync the 10M keys already pushed to it, though it does have newly added keys.

**Comment by carlosefr** [comment]
The master will not replicate keys older than the replication timestamp specified by the slave. Do you have an `.rts` file leftover from a previous run?

Otherwise, please specify what command line parameters are being used for both master and slave.

**Comment by the100rabh** [comment]
Hi @carlosefr , thanks for the reply. 

I finally figured out the issue. Here is what was happening, the database is not replicated, only update logs are sent to slaves. This updatelog has a fixed size and does not grow beyond what was specified while starting the master. 

Because of this, only the recent keys were getting udpated.( I had cleaned up the updates folder sometime thinking that the cause of this issue)

Here is my work around 
1. Save the server timestamp somewhere
2. Copy the .kch file to the new slave. 
3. Create a new .rpl file with that server timestamp that we saved in step 1.
4.  Start he new slave giving it the location of kch and rpl files in the start parameters.

Voila after sometime your server will be replicated and available for use.
